net/net.sh

```diff
@@ -64,12 +67,14 @@ updateNodes=false
 customPrograms=
 updateManifestKeypairFile=
 updateDownloadUrl=
+nBenchTpsClients=0
```

Comment:
nit: `numBenchTpsClients`.   `nBenchTpsClients` looks too much like Hungarian notation to me